panel/template/fast/list/default.css

```diff
@@ -1,9 +1 @@
-/* fast default.css */
-#header {
-  box-shadow: 2px 2px 10px silver;
-}
-
-#sidebar {
-  background-color: white;
-  box-shadow: 2px 2px 10px silver;
-}
\ No newline at end of file
+/* fast default.css */
```

Comment:
Empty file?!

Reply:
The file has been there for a long time. But more and more functionality has been moved to the template to make it dynamic.

I thought it should stay there as all template have these files ??

What should I do?

Reply:
Let's leave it for now.